datacube/drivers/postgis/_schema.py

```diff
@@ -154,19 +154,22 @@ class DatasetLocation:
 
 
 @orm_registry.mapped
-class DatasetSource:
+class DatasetLineage:
```

Comment:
👍 "source" was always confusing to me.

Reply:
:) Yeah that relation had to change.  

Now that the API can trace relationships in both directions, you just can't get away with a generic name for the relationship that implies a direction, it's too damned confusing.